uint8 id_box
uint8 id_agent
